components/script/dom/cssstylesheet.rs

```diff
@@ -36,6 +38,7 @@ impl CSSStyleSheet {
             owner: JS::from_ref(owner),
             rulelist: MutNullableJS::new(None),
             style_stylesheet: stylesheet,
+            origin_clean: Cell::new(false),
```

Comment:
Oh, you are right. I thought it will always enter process_response. (I don't know why I thought like this :) )